lib/puppet_x/puppetlabs/compatible_powershell_version.rb

```diff
@@ -0,0 +1,51 @@
+require File.join(File.dirname(__FILE__), 'powershell_version')
+
+module PuppetX
+  module PuppetLabs
+    module Dsc
+      class CompatiblePowerShellVersion
+        def self.compatible_version?
```

Comment:
As we were discussing in HipChat ... probably want to mod the implementation of `self.compatible_version?` to something as simple as:

```ruby
def self.compatible_version?
  powershell_version = PuppetX::PuppetLabs::Dsc::PowerShellVersion.version

  return false if powershell_version.nil?

  Gem::Version.new(powershell_version) >= Gem::Version.new(5)
end
```